lib/presentation/pages/tip_information_page.dart

```diff
@@ -4,11 +4,18 @@ import 'package:infinite_horizons/presentation/atoms/atoms.dart';
 import 'package:infinite_horizons/presentation/molecules/molecules.dart';
 import 'package:url_launcher/url_launcher.dart';
 
-class TipInformationPage extends StatelessWidget {
-  const TipInformationPage(this.tip);
-
+class TipInformationPage extends StatefulWidget {
```

Comment:
Why it got changed to StatefulWidget?, there is no reason for that.
Please revert this change

Reply:
My bad, I was trying another implementation earlier and forgot to revert this